app/controllers/carts_controller.rb

```diff
@@ -1,32 +1,16 @@
-# @todo Rename
+# @todo Eventually, delete
```

Comment:
I think we can delete this controller now and do the redirection in `routes.rb`:

http://guides.rubyonrails.org/routing.html#redirection